generator.ts

```diff
@@ -190,6 +212,10 @@ async function writeSectionData(reportPath: string, name: string, settings: any,
 async function writeReport(reportPath: string, report: ReportConfig, projectData: ProjectData, contents: string) {
     fs.writeFileSync(path.join(reportPath, "report.md"), contents);
     fs.writeFileSync(path.join(reportPath, "data.json"), JSON.stringify(projectData, null, 2));
+    let latest = path.join(reportPath, "..", "latest");
+    console.log(`creating symbolic link: ${reportPath} ${latest}`);
+    console.log(cp.exec(`unlink "${latest}"`).toString());
```

Comment:
Should this be `execSync` ?

```suggestion
    console.log(cp.execSync(`unlink "${latest}"`).toString());
```